Platform/TigerlakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -117,7 +168,11 @@ BoardInit (
 
   switch (InitPhase) {
   case PostTempRamInit:
-    DisableWatchDogTimer ();
+    if (PcdGetBool (PcdSblResiliencyEnabled) && PcdGetBool (PcdTopSwapBuiltForResiliency)) {
+      SetupWatchDogTimer ();
+    } else {
+      DisableWatchDogTimer ();
```

Comment:
Want to disable TCO timer in debug mode

Reply:
Will need to uncomment the condition in Stage1A.c when POC gets converted to production code.